var/spack/repos/builtin/packages/openspeedshop-utils/arm.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/plugins/views/iot/iot_view.cxx b/plugins/views/iot/iot_view.cxx
+index f4cdf48..ef1b81f 100644
+--- a/plugins/views/iot/iot_view.cxx
++++ b/plugins/views/iot/iot_view.cxx
+@@ -45,6 +45,20 @@ int64_t application_elapsed_time = 0.0;
+ # endif
+ #endif
+ /* End part 2 of 2 for Hack to get around inconsistent syscall definitions */
```

Comment:
Please remove this comment, it appears to be incorrect.

Reply:
I'm removed comment in line 5.

Reply:
I was referring to line 8

Reply:
The comment is included original code.
Please look at following url line 47:
https://github.com/OpenSpeedShop/openspeedshop/blob/master/plugins/views/iot/iot_view.cxx